### ISSUE TITLE
Load Template crash Blender 4.2 RC

### ISSUE THREAD

**Issue by jellb** [poly-hammer/BlenderTools#39]
Blender 4.2 RC crash on Load Template in Settings Dialog

**Comment by JoshQuake** [comment]
Interesting. Thanks for the report!

**Comment by JoshQuake** [comment]
I'm unable to replicate the issue. Can you please attach a zip of the blend and the template being loaded?

**Comment by youthatninja** [comment]
Yeah, I think the problem is in my user prefs after migrating from 3.6....
I have the same crashers if I do reload scripts or load factory settings in Blender

**Comment by youthatninja** [comment]
also found this, https://projects.blender.org/blender/blender/issues/123698
ue2rigify uses legasy undo

**Comment by JoshQuake** [comment]
Confirmed crashing with access violation using the above setting
[blender.crash.txt](https://github.com/user-attachments/files/16339724/blender.crash.txt)

**Comment by JoshQuake** [comment]
Welp I put break points all over the place where the file browser gets opened and none get triggered. My current theory is Blender is trying to interact with the settings dialog through context or something which ends up not existing in memory anymore since the area gets replaced with the file browser. Might be something to report to Blender themselves.

**Comment by youthatninja** [comment]
I haven't spotted any new report about the maximized area, plus I have no issues in other addons that use file browser

**Comment by JoshQuake** [comment]
Hmmm can I please get the name of some of those addons so I can test? I wonder if it's something to do with ImportHelper / ExportHelper

**Comment by youthatninja** [comment]
Pie menu Editor - Command - External Script
![image](https://github.com/user-attachments/assets/a5fa2dba-2304-45ff-acbb-2e8de4cabfed)
UVpackMaster - load preset
![image](https://github.com/user-attachments/assets/cb153bb4-e312-4778-a8ea-e69ff2c4d07f)

**Comment by youthatninja** [comment]
Heh, there is no crash on Pipline -Import -Import Asset

**Comment by JoshQuake** [comment]
Yeah that's why I'm thinking it's something in the backend having to do with trying to close out of a popup. The only difference between them from what I can see is that ImportAsset isn't located in a dialog. I'm going to try adding that operator to the dialog and see what happens.

**Comment by JoshQuake** [comment]
Yep it crashes from the dialog

**Comment by JoshQuake** [comment]
I wrote a simple addon that replicates how send2ue invokes the setting dialog and load/save template ImportHelper operators and can confirm it crashes.

```
import bpy
from bpy_extras.io_utils import ImportHelper
from bpy.types import Operator, Menu

class ExamplePanelOperator(bpy.types.Operator):
    bl_idname = "object.example_panel_operator"
    bl_label = "Example Panel Operator"
    bl_options = {'REGISTER', 'UNDO'}

    def execute(self, context):
        return context.window_manager.invoke_props_dialog(self, width=200)

    def draw(self, context):
        layout = self.layout
        layout.operator("object.example_operator")

class ExampleOperator(Operator, ImportHelper):
    bl_idname = "object.example_operator"
    bl_label = "Open File"

    def execute(self, context):
        print("You clicked on the panel operator and a file browser was opened.")
        return {'FINISHED'}

class ExampleMenu(Menu):
    bl_idname = "TOPBAR_MT_CustomMenu"
    bl_label = "Custom Menu"

    def draw(self, context):
        layout = self.layout
        layout.operator("object.example_panel_operator")

def menu_draw_func(self, context):
    self.layout.menu(ExampleMenu.bl_idname)

def register():
    bpy.utils.register_class(ExampleOperator)
    bpy.utils.register_class(ExamplePanelOperator)
    bpy.utils.register_class(ExampleMenu)
    bpy.types.TOPBAR_MT_editor_menus.append(menu_draw_func)

def unregister():
    bpy.types.TOPBAR_MT_editor_menus.remove(menu_draw_func)
    bpy.utils.unregister_class(ExampleMenu)
    bpy.utils.unregister_class(ExamplePanelOperator)
    bpy.utils.unregister_class(ExampleOperator)

if __name__ == "__main__":
    register()
```

**Comment by JoshQuake** [comment]
I have created a bug report for Blender [here](https://projects.blender.org/blender/blender/issues/125589)

**Comment by JoshQuake** [comment]
Updating with latest activity from Blender. Still no fix, but at least they are aware and trying to come up with solutions.

https://projects.blender.org/blender/blender/pulls/132041
